lib/tasks/message_queue.rake

```diff
@@ -0,0 +1,19 @@
+namespace :message_queue do
+  desc "Index documents that are published to the publishing-api"
+  task :index_documents_from_publishing_api do
+    ENV["RABBITMQ_HOSTS"] = "localhost"
```

Comment:
I think these should be set in puppet, rather than in the app; puppet is the place that we normally put configuration.

Also, this value for RABBITMQ_HOSTS won't be right in production.